### PR TITLE
chore: push NUnit as NuGet

### PR DESCRIPTION
**Pull request by avodovnik**
- rename the namespace back to NUnit
- introduce a common version props file
- enable signing & strong name in the new project
- add the new project to pipeline YML

### CODE REVIEW

File: src/Common/Version.props [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<Project>
+  <PropertyGroup>
+    <AssemblyVersion>1.11.1</AssemblyVersion>
```
Comment (kblok) [inline src/Common/Version.props:3]: I think we don't need to keep those two projects in sync. We might need to update Nunit only when we change something there or there is a breaking change in PW.

File: src/Playwright.NUnitTest/Playwright.NUnit.csproj [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    <AssemblyName>Microsoft.Playwright.NUnit</AssemblyName>
+    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
+    <IsPackable>true</IsPackable>
+    <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
+  </PropertyGroup>
+  <Import Project="../Common/Version.props" />
+  <Import Project="../Common/SignAssembly.props" />
+  <ItemGroup>
+    <ProjectReference Include="..\Playwright\Playwright.csproj" />
```
Comment (kblok) [inline src/Playwright.NUnitTest/Playwright.NUnit.csproj:27]: One think we should do is declaring this as a nuget dependency.

**Comment by pavelfeldman** [comment]
I called it NUnitTest originally because when I used NUnit.Foo in the code, compiler yelled at me for clash between NUnit.Foo and Microsoft.Playwright.NUnit.Foo, effectively making the use of the full-qualified NUnit.Foo impossible. Was that a glitch on my side?

**Comment by avodovnik** [comment]
@pavelfeldman Yes, it yelled at me too. You can tell it to be [quiet](https://github.com/microsoft/playwright-sharp/pull/1473/files#diff-5a8526cb33dc7409785f36da0c6a464bca2829cac0fa48a8767801ce11e8402aR30) by using `global::` :-). 

Having a `.NUnit` suffix seems more natural to me. There are also packages that do the same, like [SpecFlow](https://www.nuget.org/packages/SpecFlow.NUnit/), I don't want to call it "Test" because it implies it's an actual _test_. 

I'd rather we keep it `.NUnit`. The clash, like above is a common thing, and it's usually not a problem. But, if we wanted to play it safe, we could call ourselves `Microsoft.Playwright.NUnitExtensions` or something to that tune.

File: src/Playwright.NUnitTest/Playwright.NUnit.csproj (hunk 2 above)
Comment (avodovnik) [reply src/Playwright.NUnitTest/Playwright.NUnit.csproj:27]: That happens automatically apparently. Looking at the produced nuget package:

image

File: src/Common/Version.props (hunk 1 above)
Comment (avodovnik) [reply src/Common/Version.props:3]: Yes, that was my thought too, *but* it has a dependency on Playwright. While it's technically a `>=` dependency, it felt like keeping them in-sync would be the right bet.

**Review by pavelfeldman — APPROVED** [review]

**Pull request merged**